Backend/Models/Messages/CommandMessage.cs

```diff
@@ -136,16 +136,13 @@ private void ProcessMute()
         }
         private void ProcessRemember()
         {
-            int param1, param2;
-            if (!int.TryParse(_from, out param1) || !int.TryParse(_to, out param2))
+            int fromId, toId;
```

Comment:
Any  LSMP changes needed to reflect this decision?

Reply:
Nope, all good